_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,150 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks existence of role(s) on required owned elements.
+
+success_criterion:
+- 1.3.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) with a [non empty](#non-empty) `role` attribute whose value is applicable to required owned elements.
+
+**Note:** I
+
+### Expectation
+
+Each target element has atleast one owned element with a relevant `role` attribute.
+
+## Assumptions
+
+*There are currently no assumptions/ add any assumptions as necessary*
+
+## Accessibility Support
+
+*There are no major accessibility support issues known for this rule/ or add any support details as necessary*
+
+## Background
+
+- [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
+
+## Test Cases
+
+### Passed
+
+#### Pass example 1
+
+Owned `ul` element with role `list` has `span` element with role `listitem`.
```

Comment:
Here I would say that the `span` role `lisitem` has a required owned element of `list` and `group`